app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -386,10 +297,6 @@ public void onResume() {
 
         boolean isSettingsChanged = store.getBoolean(Prefs.IS_CONTRIBUTION_COUNT_CHANGED, false);
```

Comment:
this could be deleted now, right?

Reply:
IS_CONTRIBUTION_COUNT_CHANGED is now only used to put the boolean, never retrieve it, can be deleted